### PR TITLE
Document topologies in parameters.md

### LINKED ISSUE #1701: Please document allowedTopologies
/kind bug

With #1263, what little documentation there was of `allowedTopologies` was removed. Please document how to use it. In particular:

- what the effect is
- what happens if it is not specified
- what Topology Key to use and what needs to happen to make it work. (See #729, #1360, #912)

### PR DESCRIPTION
**Pull request by ConnorJC3**
**Is this a bug fix or adding new feature?**

Documentation 

**What is this PR about? / Why do we need it?**

Adds topology documentation (along with minor cleanup to fit it in) to `parameters.md`
Fixes #1701 

**What testing is done?** 

N/A, docs change

### CODE REVIEW

File: docs/parameters.md [hunk 1]
```diff
@@ -31,3 +31,26 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | io2 (blockExpress = false) | 100            | 64000         | 500               |
 | io2 (blockExpress = true)  | 100            | 256000        | 500               |
 | gp3                        | 3000           | 16000         | 500               |
+
+## Volume Availability Zone and Topologies
+
+The EBS CSI Driver supports the [`WaitForFirstConsumer` volume binding mode in Kubernetes](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode). When using `WaitForFirstConsumer` binding mode the volume will automatically be created in the appropriate Availability Zone and with the appropriate topology. The `WaitForFirstConsumer` binding mode is recommended whenever possible for dynamic provisioning.
+
+When using static provisioning, or if `WaitForFirstConsumer` is not suitable for a specific usecase, the Availability Zone can be specified via the standard CSI toplogy mechanisms. The EBS CSI Driver supporsts both the key `topology.kubernetes.io/zone` or the key`topology.ebs.csi.aws.com/zone` to speicify the Availability Zone.
```
Comment (AndrewSirenko) [inline docs/parameters.md:39]: speicify -> specify

**Review by AndrewSirenko — CHANGES_REQUESTED** [review]

File: docs/parameters.md (hunk 1 above)
Comment (ConnorJC3) [reply docs/parameters.md]: Fixed that and also spotted a missing space on the same line that I just fixed

Comment (AndrewSirenko) [reply docs/parameters.md]: supporsts  -> supports

File: docs/parameters.md [hunk 2]
```diff
@@ -31,3 +31,26 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | io2 (blockExpress = false) | 100            | 64000         | 500               |
 | io2 (blockExpress = true)  | 100            | 256000        | 500               |
 | gp3                        | 3000           | 16000         | 500               |
+
+## Volume Availability Zone and Topologies
+
+The EBS CSI Driver supports the [`WaitForFirstConsumer` volume binding mode in Kubernetes](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode). When using `WaitForFirstConsumer` binding mode the volume will automatically be created in the appropriate Availability Zone and with the appropriate topology. The `WaitForFirstConsumer` binding mode is recommended whenever possible for dynamic provisioning.
+
+When using static provisioning, or if `WaitForFirstConsumer` is not suitable for a specific usecase, the Availability Zone can be specified via the standard CSI toplogy mechanisms. The EBS CSI Driver supporsts both the key `topology.kubernetes.io/zone` or the key `topology.ebs.csi.aws.com/zone` to specify the Availability Zone.
```
Comment (AndrewSirenko) [inline docs/parameters.md:39]: supporsts -> supports

**Comment by AndrewSirenko** [comment]
Love the examples and links. This is great. The more we teach, the happier our users.

File: docs/parameters.md [hunk 3]
```diff
@@ -31,3 +31,26 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | io2 (blockExpress = false) | 100            | 64000         | 500               |
 | io2 (blockExpress = true)  | 100            | 256000        | 500               |
 | gp3                        | 3000           | 16000         | 500               |
+
+## Volume Availability Zone and Topologies
+
+The EBS CSI Driver supports the [`WaitForFirstConsumer` volume binding mode in Kubernetes](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode). When using `WaitForFirstConsumer` binding mode the volume will automatically be created in the appropriate Availability Zone and with the appropriate topology. The `WaitForFirstConsumer` binding mode is recommended whenever possible for dynamic provisioning.
+
+When using static provisioning, or if `WaitForFirstConsumer` is not suitable for a specific usecase, the Availability Zone can be specified via the standard CSI toplogy mechanisms. The EBS CSI Driver supports specifying the Availability Zone via either the key `topology.kubernetes.io/zone` or the key `topology.ebs.csi.aws.com/zone`.
```
Comment (torredil) [inline docs/parameters.md:39]: nit: `toplogy` -> `topology`.
/lgtm

**Review by torredil — APPROVED** [review]

**Review by AndrewSirenko — APPROVED** [review]

File: docs/parameters.md (hunk 3 above)
Comment (ConnorJC3) [reply docs/parameters.md]: fixed

**Pull request merged**